test/pythontest_skeleton.py

```diff
@@ -0,0 +1,15 @@
+#Created by Himanshu Nachane (@Himanshu1495)
```

Comment:
We need pythontest_skeleton so that new Python contributors can easily use it to create their tests. All they need to do is remove the comments in the file and replace it with actual data.